### DISCUSSION TITLE
Recent Updates and Docker Image Question

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#4469] [Q&A] [answered]
Just wanted to check in as I have seen a few updates to the Edge/Dev code over the past week or two, but the docker images have not been updated in 16+ days.  I'm using EDGE-ARM images on  Raspberry Pi.  Often I see the updates in my RSS feed and I'm usually able to do a docker pull of the updated image shortly after that.  

Maybe the recent updates have not impacted the docker builds, but just making sure that was not some issue with these docker images being automatically created and uploaded to the hub.  

Again, thank you all for the work you put into this project.

**Comment by Frenzie** [comment]
They are automated, so that's certainly weird.

**Reply by quella** [reply]
I assumed there was some automation, but on the hub for Docker, you can see that they are not being updated.  Again, more a heads up in the event something broke in this process.

**Comment by Alkarex** [comment, marked as answer]
Thanks for the heads up. Our Docker builds are normally automated, but our open source agreement has to be renewed every year. I filed the official form on 4th July but it visibly hasn't been processed yet...
In the meantime, I have just pushed some new images for our various edge variants. (edit: done)

**Reply by quella** [reply]
Thank you both, and I simply wanted to make you aware of a potential issue.  Glad to hear it will be resolved shortly, and thank you for pushing out some manual updates as well.  Always like a few updates!

**Reply by Alkarex** [reply]
Now our Docker Automated Builds have been restored, so everything should be back to normal :-)
![image](https://user-images.githubusercontent.com/1008324/185761915-a311e676-e64e-498d-97ac-b48d11cc5349.png)

**Comment by quella** [comment]
Thanks for the heads up and thank you for your manual building for a bit.  I will keep the new releases coming.
